apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/CharSequenceToContainCreatorSamples.kt

```diff
@@ -0,0 +1,257 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.expect
+import kotlin.test.Test
+
+class CharSequenceToContainCreatorSamples {
+
+    @Test
+    fun value() {
+        expect("ABC") toContain o exactly 1 value "A"
```

Comment:
You missed to copy the comment (also in other functions below), see ch.tutteli.atrium.api.fluent.en_GB.samples.CharSequenceToContainCreatorSamples
```
// `value` is a final step in the CharSequence.toContain expectation-building process and can be used with
// various checkers (see CharSequenceToContainCheckerSamples)
```